### ISSUE TITLE
Ansible Tower Survey enhancements

### ISSUE THREAD

**Issue by brsanche** [ansible/awx#11650]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
This is an issue to request a change to the Ansible Survey options. It would be much more user friendly to have the option to create user friendly variables and then have that translated to variable in the background. E.g. days of the week. I want the number of the day to be presented to my playbook but my user to see the day of the week as a word in a drop down, e.g. Monday, Tuesday, Wednesday etc.

An example of a playbook would be as follows, the below Ansible module will be used to find files with a date in their name.
The Ansible Tower template has a survey which collects the day of the month, Month of the year and the year with numbers.
I want to display the months to the user as Jan, Feb, March, April, etc etc to make it easier to use. However the file names contain numbers., e.g. /var/log/product/name-of-log-2021-12-28.log

We want the user to be able to chose 28 from the day box, December from the month box and 2021 from the year boxes within the survey in Tower.

Currently, if I want to have this, I must write something like this.
 
```
---
- name: playbook to find logs
   hosts: all
   tasks:
   - name: set month word to number
      set_fact:
         log_month_number: 12
      when: ( log_month_word == "December" )

   - name: find logs
      find:
         paths: /var/log/
         pattern: "{{ log_day_number }}-{{ log_month_number }}-{{ log_year_number }}"{code}
```

Basically, I want the user to be able to select a "pretty" answer from a dropdown and then Tower have the option to set a value to this answer i.e. user selects December and tower injects 12 as the variable.
